tests/integration/test_charm.py

```diff
@@ -75,7 +75,6 @@ async def test_relate_dependencies(ops_test: OpsTest):
     await ops_test.model.add_relation("istio-pilot:ingress", "kubeflow-dashboard:ingress")
     await ops_test.model.add_relation("istio-pilot", "kubeflow-volumes")
     await ops_test.model.wait_for_idle(
-        wait_for_units=True,
```

Comment:
the parameter was deprecated in pyjuju 3.0, in this case it will still be set to true inside the method so the behavior has not changed, see [method](https://github.com/juju/python-libjuju/blob/master/juju/model.py#L2630)

Reply:
It looks like it's a number now though, like "wait for this number of units to become available". I don't think we need this in any case.